WordPress/wordpress-playground
# Browser: Disable the network support by default and add a UI control to enable it

## What is this PR doing?

This PR disables the network support by default and adds a UI control to enable it.

Why?

https://github.com/WordPress/wordpress-playground/pull/724 introduced support for wp_safe_remote_get() request and enabled it by default on playground.wordpress.net. The problem is, all the requests block rendering of WordPress pages and noticeably slow down the site. Let's disable it by default for a lightweight user experience, and then add an easy way to enable it, for example in the configuration modal.

Closes #755

## Testing Instructions

* Go to http://localhost:5400/website-server/?url=/wp-admin/plugin-install.php
* Confirm the plugins aren't loaded and you see a communicative error message
* Go to the settings modal
* Enable the network support
* Confirm the plugins are loaded now

<img width="1121" alt="CleanShot 2023-11-27 at 17 10 45@2x" src="https://github.com/WordPress/wordpress-playground/assets/205419/ea95b783-d7a1-45c6-ab95-90b5c8ec6ce4">
